posts/2020/12/jupyterlab-winter-theme.md

```diff
@@ -0,0 +1,233 @@
+<!--
+.. title: JupyterLab Winter Theme
+.. slug: jupyterlab-winder-theme
+.. date: 2020-12-01 09:00:00 UTC-00:00
+.. author: Matthias Bussonnier, Isabela Presedo Floyd, Eric Charles
+.. tags: Labs, Jupyter, Theme, JupyterLab
```

Comment:
I would like to add a tag we decide on for this series so it's easier for people to find them in one place. I've proposed `JupyterTutorials` as a solution for now.

```suggestion
.. tags: Labs, Jupyter, Theme, JupyterLab, JupyterTutorials
```